documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -26,6 +32,18 @@ alias samm='java -jar /location/to/samm-cli-{esmf-sdk-version}.jar'
 ----
 ====
 
+*For the Windows native binary:*
+. *Download*: download above or visit our github https://github.com/eclipse-esmf/esmf-sdk/releases[releases page] and download executable file for Windows.
+. *Extract*: extract it to a location of your choice.
+. *Open Command Prompt*: Launch the Command Prompt and navigate to the directory (samm-cli-windows) containing our application.
+. *Run*: In the command prompt, execute the command ``./samm.exe` to start the application.
+
+*For the Linux native binary:*
+. *Download*: download above or visit our github https://github.com/eclipse-esmf/esmf-sdk/releases[releases page] and download executable file for Linux.
+. *Extract*: Unpack the archive to your preferred directory.
+. *Open Terminal*: Open the Terminal and navigate to the directory with the application.
+. *Execute*: Run the application using the command `./samm`.
+
 Successful execution of a command is signaled by returning 0. In case of logical or some other internal problem error code 1 is returned.
```

Comment:
In case of a logical or other internal error the error code 1 is being returned.